solr-search/src/main/java/edu/unc/lib/dl/search/solr/model/CutoffFacet.java

```diff
@@ -145,7 +145,7 @@ public String getDisplayValue() {
 		CutoffFacetNode lastNode = this.getHighestTierNode();
 		if (lastNode == null)
 			return null;
-		return lastNode.getDisplayValue();
+		return lastNode.getSearchKey();
```

Comment:
CutoffFacets don't have a display value anymore, that was previously that label.  However, I should replace the contents of this method with a call to CutoffFacet.getSearchKey instead of reimplementing it.